frontend/src/containers/Search/Background.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { css, StyleSheet } from 'aphrodite';
+
+export default () => (
+  <img src="http://lorempixel.com/1000/600/abstract" className={css(styles.img)} alt="background" />
```

Comment:
I find this very busy, maybe I'm just old ^^   It took a few seconds to load and I think we have a more minimalist style currently? just my two pennies.

Reply:
Yes, it was just for fun. I will see if there is some good image hosting CDN or we may save the background image in our server.